src/RelayClient.ts

```diff
@@ -83,13 +81,20 @@ class RelayClient extends EnvelopingEventEmitter {
 
   private readonly _httpClient: HttpClient;
 
-  constructor(httpClient: HttpClient = new HttpClient()) {
+  constructor(
+    httpClient: HttpClient = new HttpClient(),
+    private _relayServerUrl?: string
```

Comment:
What's the purpose of having the additional property here? I'm trying to track the property usage, but IMHO it would be the same if the client was configured to with a single server only.